release-notes/opensearch-release-notes-1.3.18.md

```diff
@@ -0,0 +1,16 @@
+# OpenSearch and OpenSearch Dashboards 1.3.18 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 1.3.18](https://opensearch.org/versions/opensearch-1-3-18.html) includes the following maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.18.md).
```

Comment:
Missed OSD releasenotes.

Reply:
There is no commits update from OSD since 1.3.17.